docs/monitoring/inventory.md

```diff
@@ -46,21 +43,61 @@ In addition, an overview of all the installed packages on a given system is avai
 - Package name
 - Package version
 
-![Inventory Overview](../images/monitoring/inventory_packages.png)
+#### Comparing Hosts
+
+The Inventory UI allows comparing different systems to quickly and efficiently see differences between them. Use filtering to see differences in specific packages.
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://cdn.sematext.com/videos/sematext-infra-inventory-hosts-compare.mp4" type="video/mp4" />
+</video>
+
+#### Grouping Hosts By Installed Packages
+
+Hosts can be filtered and grouped by tags. One of the available tags for grouping is installed.packagas tag, which will put servers with exactly the same packages installed in the same group. This can be useful to quickly check if all servers that should have the same packages installed are indeed the same.
```

Comment:
_One of the available tags for grouping is installed.packagas tag,_

Something is off here?